tests/checkbox/checkbox-provider-tdx/units/basic/jobs.pxu

```diff
@@ -83,6 +83,18 @@ command:
   export LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu/:$LD_LIBRARY_PATH
   setup-env-and-run test_boot_multiple_vms.py
 
+id: tdx-basic/td-stress-boot
+category_id: tdx-basic
+flags: simple
+_summary: Test boot stress
+depends:
+after:
+requires:
+  executable.name == 'qemu-system-x86_64'
```

Comment:
Just curious how this will handle our patched, tdx-enabled QEMU? I guess since checkbox runs as a snap that it's up to us to build a snap with the custom QEMU?

Reply:
the tests are run against a host system components (here : kernel, qemu), for this kind of testing, no need to embed qemu inside the snap

Reply:
ok, so the tests depend on the tdx-enabled versions being installed on the host system?